tests/acceptance/products-test.js

```diff
@@ -0,0 +1,61 @@
+import { describe, it, beforeEach, afterEach } from 'mocha';
+import { expect } from 'chai';
+import startApp from 'bsc-ember/tests/helpers/start-app';
+import destroyApp from 'bsc-ember/tests/helpers/destroy-app';
+import page from 'bsc-ember/tests/pages/product';
+
+describe('Acceptance | products', function() {
+  this.timeout(0);
+  let application;
+
+  beforeEach(function() {
+    application = startApp();
+  });
+
+  afterEach(function() {
+    destroyApp(application);
+  });
+
+  it('can visit products page and go to product detail page', async function() {
+    server.create('product', { name: 'Personal Computer' });
+    await page.visit();
+
+    // show 
+    await page.products(0).open();
+    expect(page.productName).to.include('Personal Computer');
+
+    // edit
+    await page.openProducts()
+      .openProductEdit()
+      .editProductName('Apple 1')
+      .saveProduct();
+    expect(server.db.products[0].name).to.eq('Apple 1');
```

Comment:
consider sometimes it is good to test both

a) what user see (this is one that would be nice to add here)
b) is the change saved on the backend